Wifi: Add wifi doc for mxv,CT, T30(fr) (#5)
hostmobility/documentation: docs/interfaces/wifi.md

---
title: WiFi
tags:
    - MX-V
    - T30
    - T30 FR
---
## Overview

> For more help you can go to this link [wiki.archlinux.org](https://wiki.archlinux.org/title/Network_configuration/Wireless)

Reviewer: ```suggestion
This page briefly explains how to scan for and connect to WiFi networks using wpa_supplicant.
For more information, see the [Arch Linux Wiki](https://wiki.archlinux.org/title/Network_configuration/Wireless).
```

## Platform specific
- [MX-V](mxv/wifi.md)
- [T30](mx4/wifi.md)
- [T30 FR](mx4/wifi.md)

> ## Example 

Reviewer: ```suggestion
## Examples
```

> ### scan wifi networks

Reviewer: ```suggestion
### Scan for WiFi networks
```
